src/main/java/com/epam/test_generator/controllers/version/caze/PropertyDifferenceTransformer.java

```diff
@@ -23,4 +30,46 @@ public PropertyDifferenceDTO toDto(PropertyDifference prop) {
             .map(this::toDto)
             .collect(Collectors.toList());
     }
+
+    public Object convert(Object sourceFieldValue) {
+        if (sourceFieldValue == null) {
+            return null;
+        }
+
+        if (sourceFieldValue instanceof Step) {
+            Step source = (Step) sourceFieldValue;
+            return convertStepToDTO(source);
+        } else if (sourceFieldValue instanceof Tag) {
+            Tag source = (Tag) sourceFieldValue;
+            return convertTagToDTO(source);
+        } else if (sourceFieldValue instanceof Case) {
+            Case source = (Case) sourceFieldValue;
+            return convertCaseToDTO(source);
+        } else if (sourceFieldValue instanceof ZonedDateTime) {
+            return String.valueOf(((ZonedDateTime) sourceFieldValue).toInstant().toEpochMilli());
```

Comment:
Fixed

Reply:
I returned back to  String.valueOf(((ZonedDateTime) sourceFieldValue).toInstant().getEpochSecond()) because this is correct behavior when we get time value from DB